pycc/local.py

```diff
@@ -49,7 +49,8 @@ class Local(object):
     _build_PAO(): build PAO orbital rotation tensors
     _build_PNO(): build PNO orbital rotation tensors   
     _build_PNOpp(): build PNO++ orbital rotation tensors
-    
+    _trans_integral(): transform Fock matrix and ERI from the MO basis to a local basis 
```

Comment:
Why the leading underscore?  Also, the function name has "integrals" rather than "integral".